google/cloud/security/inventory/inventory_loader.py

```diff
@@ -185,8 +185,43 @@ def _run_pipelines(pipelines):
                          exc_info=True)
             pipeline.status = 'FAILURE'
         run_statuses.append(pipeline.status == 'SUCCESS')
+    _postprocess_statuses(pipelines, run_statuses)
     return run_statuses
 
+def _postprocess_statuses(pipelines, run_statuses):
```

Comment:
This method is very specific to processing group and group members.  So, I would suggest to change it to:
```
def _adjust_group_member_run_status_if_necessary(pipelines, run_statuses)
```

Reply:
That's a bit long :) but can do. I guess the idea was that the current `_postprocess_statuses()` implementation is just a first step but maybe we will need to add more to it later.

Reply:
I think this method should be renamed.  If there are any other postprocess adjustments to be in separate helper methods.

Reply:
Looking for a short name, pylint is complaining :-P

    C:191, 0: Function name "_adjust_group_members_run_status" doesn't conform to u'[a-z_][a-z0-9_]{2,30}$' pattern (invalid-name)

Reply:
Done with all changes. Please take a look. Thanks!